### PR TITLE
systemctl start and enable

### PR DESCRIPTION
**Pull request by exarkun**
This improves the installation documentation for CentOS 7.

### CODE REVIEW

File: docs/installation/centos.md [hunk 1]
```diff
@@ -58,9 +58,10 @@ There are two ways to install Docker Engine.  You can use `curl` with the  `get.
 		
 		This script adds the `docker.repo` repository and installs Docker.
 
-4. Start the Docker daemon.
+4. Start the Docker daemon and enable it so it starts on future boots.
 
-		$ sudo service docker start
+		$ sudo systemctl start docker
+                $ sudo systemctl enable docker
```
Comment (hqhq) [inline docs/installation/centos.md]: Is this not aligned?

**Comment by hqhq** [comment]
And you should add: `Fixes: #15456` in changelog, so the original issue would be closed once this is merged.

**Comment by moxiegirl** [comment]
@exarkun There is a small change on the alignment pointed out by @hqhq would you like to make that change or shall I carry for you?

**Comment by calavera** [comment]
@exarkun can you fix that small misalignment?

**Comment by thaJeztah** [comment]
Carried in https://github.com/docker/docker/issues/15967

**Pull request closed without merging**